checkov/sca_image/runner.py

```diff
@@ -174,11 +174,12 @@ def iterate_image_files(self, abs_fname: str, report: Report, runner_filter: Run
             if image_referencer.is_workflow_file(abs_fname):
                 images = image_referencer.get_images(file_path=abs_fname)
                 for image in images:
-                    if not strtobool(os.getenv('CHECKOV_PRESENT_CACHED_RESULTS', "False")):
+                    if strtobool(os.getenv('CKV_CREATE_SCA_IMAGE_REPORTS_FOR_IR', "True")):
```

Comment:
Probably a nit, but is `CKV_CREATE_SCA_IMAGE_REPORTS_FOR_IR` supposed to be True by default?

If we want to allow this functionality by default then I think the env var name should express the negation of the functionality, something like - `CHECKOV_NO_SCA_IMAGE_REPORTS_FOR_IR`, if that makes sense

Reply:
I know what you mean, but I don't think there is any issue with having an env var which is by default enabled, we also do it in other areas. It can also become confusing that to similar env vars needs to be configured differently. And lastly they are not really meant to be used by normal users 🙂

Reply:
My thought was like Anton's, that since these two env vars come as a logical pair, it'd be clearer to have them in the same direction (both FF that enable a feature). And yes, the first by default is on and the second is off.